soh/soh/Enhancements/randomizer/hint.h

```diff
@@ -8,24 +8,26 @@ class Hint {
     public:
     Hint();
     Hint(Text text_);
-    Hint(Text text_, RandomizerCheck hintedLocation_, HintType hintType_, Text hintedRegion_);
+    Hint(Text text_, RandomizerCheck hintedLocation_, HintType hintType_, std::string distributionName_, RandomizerArea hintedArea_);
     const Text& GetText() const;
     RandomizerCheck GetHintedLocation();
     void SetHintedLocation (RandomizerCheck location);
     HintType GetHintType();
     void SetHintType (HintType type);
-    std::string GetHintedRegion();
-    const Text& GetHintedRegionText();
-    void SetHintedRegion (Text region);
+    RandomizerArea GetHintedArea();
+    void SetHintedArea (RandomizerArea area);
+    std::string GetDistribution();
+    void SetDistribution (std::string distribution);
     void ResetVariables();
-    bool IsAddedToPool();
+    bool IsAddedToPool(); //RANDOTODO what is this for
```

Comment:
I added this because I thought it was checked a few times in reference to hints back when they were actually stored as items in the itemLocation table in 3drando. If it's not being used anywhere then I must have misread something. I'd leave this one alone, it may be used later if we fully separate hint locations from item locations.